src/main/java/uk/gov/pay/directdebit/common/exception/ApiException.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.pay.directdebit.common.exception;
+
+import org.slf4j.Logger;
+import uk.gov.pay.directdebit.app.logger.PayLoggerFactory;
+
+public class ApiException extends RuntimeException {
+    private static final Logger LOGGER = PayLoggerFactory.getLogger(ApiException.class);
+    private Integer statusCode;
+
+    Integer getStatusCode() {
+        return statusCode;
```

Comment:
The status of a response does not belong to an exception but the mapper. So mapper receives an exception and decides what status code to send (an invoker of an exception shouldn't now about status codes in responses)